templates/nrpe.cfg.epp

```diff
@@ -121,7 +121,7 @@ dont_blame_nrpe=<%= $dont_blame_nrpe %>
 
 #allow_bash_command_substitution=0
 <% if $allow_bash_command_substitution { -%>
```

Comment:
It looks like the original intention was to default to whatever nrpe's default is (possibly version dependent?) by not writing anything but allowing either true/false to explicitly set the option.

Reply:
Not sure if it's important to some people to be able to explicitly set `allow_bash_command_substitution=0` (even if that has always been the default??).  Either way, I've opened https://github.com/voxpupuli/puppet-nrpe/pull/32 with just a fix to the template and the tests I should have added  when I broke everything 16 months ago! :)